themes-default/slim/views/config_postProcessing.mako

```diff
@@ -21,1627 +9,528 @@ window.app = new Vue({
-                configArray.push(seasonAllBanner ? '1' : '0');
-
-                let curNumber = 0;
-                for (let i = 0, len = configArray.length; i < len; i++) {
-                    curNumber += parseInt(configArray[i], 10);
-                }
-                if (curNumber > curMost) {
-                    curMost = curNumber;
-                    curMostProvider = generatorName;
+            let config = {
+                postProcessing: this.postProcessing,
+                metadata: {
+                    metadataProviders: this.metadataProviders
                 }
+            };
 
-                $('#' + generatorName + '_eg_show_metadata').prop('class', showMetadata ? 'enabled' : 'disabled');
-                $('#' + generatorName + '_eg_episode_metadata').prop('class', episodeMetadata ? 'enabled' : 'disabled');
-                $('#' + generatorName + '_eg_fanart').prop('class', fanart ? 'enabled' : 'disabled');
-                $('#' + generatorName + '_eg_poster').prop('class', poster ? 'enabled' : 'disabled');
-                $('#' + generatorName + '_eg_banner').prop('class', banner ? 'enabled' : 'disabled');
-                $('#' + generatorName + '_eg_episode_thumbnails').prop('class', episodeThumbnails ? 'enabled' : 'disabled');
-                $('#' + generatorName + '_eg_season_posters').prop('class', seasonPosters ? 'enabled' : 'disabled');
-                $('#' + generatorName + '_eg_season_banners').prop('class', seasonBanners ? 'enabled' : 'disabled');
-                $('#' + generatorName + '_eg_season_all_poster').prop('class', seasonAllPoster ? 'enabled' : 'disabled');
-                $('#' + generatorName + '_eg_season_all_banner').prop('class', seasonAllBanner ? 'enabled' : 'disabled');
-                $('#' + generatorName + '_data').val(configArray.join('|'));
+            $store.dispatch('setConfig', {section: 'main', config: config}).then(() => {
+                this.$snotify.success('Saved postprocessing config', 'Saved', { timeout: 5000 });
+            }).catch(error => {
+                this.$snotify.error(
+                    'Error while trying to save postprocessing config',
+                    'Error'
+                );
             });
-
-            if (curMostProvider !== '' && first) {
-                $('#metadataType option[value=' + curMostProvider + ']').prop('selected', true);
-                $(this).showHideMetadata();
-            }
-        };
-
-        $(this).refreshMetadataConfig(true);
-        $('img[title]').qtip({
-            position: {
-                at: 'bottom center',
-                my: 'top right'
-            },
-            style: {
-                tip: {
-                    corner: true,
-                    method: 'polygon'
-                },
-                classes: 'qtip-shadow qtip-dark'
+        },
+        processMethodDescription(processMethod) {
+            return this.processMethods.get(processMethod)
```

Comment:
Can this not be a computed prop?

Reply:
but you can't pass params to a computed right?

Reply:
But that function is not even being used?

You don't need this kind of solution in this case (as you already found out when you did the `v-for` on a list of objects.
But in any case - you could make a computed that returns an object:
```js
processMethodDescriptions() {
    const { processMethods } = this;
    const descriptions = {};
    processMethods.forEach(method => {
        descriptions[method.value] = method.text;
    });
    return descriptions;
}
```